src/ProductAttributePageTitle.php

```diff
@@ -20,7 +20,9 @@ public static function init() {
     }
 
     // Register ACF field.
-    ProductAttributePageTitle::register_acf_page_title($group_filter);
+    if (function_exists('register_field_group')) {
```

Comment:
@Mauricio-Urrego I think this would make more sense if we added it before it was actually invoked, like on line 33. This is only a suggestion, feel free to decide yourself, nice catch.